model/vm.rb

```diff
@@ -14,7 +14,7 @@ class Vm < Sequel::Model
   one_to_many :active_billing_records, class: :BillingRecord, key: :resource_id do |ds| ds.active end
   one_to_many :pci_devices, key: :vm_id, class: :PciDevice
   many_to_many :load_balancers, left_key: :vm_id, right_key: :load_balancer_id, join_table: :load_balancers_vms
-  one_to_many :load_balancers_vms, key: :vm_id, class: LoadBalancersVms
+  one_to_one :load_balancers_vms, key: :vm_id, class: LoadBalancersVms
```

Comment:
It's not a blocker and also I know the class name is plural. But vm has `one_to_one :load_balancers_vms` and load balancer has `one_to_many :load_balancers_vms, key: :load_balancer_id, class: LoadBalancersVms`. Their name are same but one returns an array others return a single object. It feels like counter-intuitive. Should we rename it?

Reply:
You're right but the `load_balancer.load_balancers_vms` need to return multiple items but the `vm.load_balancers_vms` will have a single item. They are both resolved through the same join table. I would like to keep it as is for now.